rules/windows/defense_evasion_code_injection_conhost.toml

```
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/08/31"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/31"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious Conhost child process which may indicate a code injection activity. Verify process details such as
+commandline, parent process and network connections.
```

```suggestion
Identifies a suspicious Conhost child process which may be an indication of code injection activity. 
```
move to `note` - Verify process details such as command line, parent process name, and network connections.